x-pack/plugins/uptime/e2e/page_objects/monitor_management.tsx

```diff
@@ -63,7 +63,7 @@ export function monitorManagementPageProvider({
     },
 
     async editMonitor() {
-      await this.clickByTestSubj('monitorManagementEditMonitor');
+      await page.click(this.byTestId('monitorManagementEditMonitor'), { delay: 800 });
```

Comment:
I've seen some recordings where clicks on icon buttons are just selecting it, not triggering click action. Adding a click delay seems to fix the issue.